onnxutils/onnxconverter_common/onnx_ops.py

```diff
@@ -497,6 +497,35 @@ def apply_scaled_tanh(scope, input_name, output_name, container, operator_name=N
         # output = a * d
         apply_mul(scope, [aName, dName], output_name, container)
 
+def apply_slice(scope, input_name, output_names, container, starts, ends,
```

Comment:
Since slice only has one output, call it output_name for consistency.

Reply:
Good point, fixed.